random-beans/src/main/java/io/github/benas/randombeans/randomizers/collection/MapRandomizer.java

```diff
@@ -49,16 +49,6 @@
     private final Randomizer<V> valueRandomizer;
 
     /**
-     * Create a new {@link MapRandomizer} with a random number of entries.
-     *
-     * @param keyRandomizer   the randomizer for keys
-     * @param valueRandomizer the randomizer for values
-     */
-    public MapRandomizer(final Randomizer<K> keyRandomizer, final Randomizer<V> valueRandomizer) {
```

Comment:
Why do you want to remove these randomizers constructors and static factory methods?